### DISCUSSION TITLE
AVG reporting freshly downloaded Windows installer as malware

### DISCUSSION THREAD

**Discussion by PaulWebster** [mitmproxy/mitmproxy#5431] [General]
Just downloaded mitmproxy 8.1 for Windows.
AVG blocked the installer when I ran it.
VirusTotal shows AVG and a small number of other AV suppliers have flagged it. I assume it is a false positive but thought it best to let you know.
I have submitted it to AVG for further analysis.

https://www.virustotal.com/gui/file/c9a20a033d97e5e978ef1b122384a25376ac338067546af994deec70d9806cab/detection

Avast - FileRepMalware
AVG - FileRepMalware
McAfee-GW-Edition - BehavesLike.Win64.Generic.vc
SecureAge APEX - Malicious
Zillya - Trojan.Disco.Win64.211

**Comment by mhils** [comment]
Thanks! This is indeed a false-positive. We use PyInstaller to bundle everything into a single executable, which unfortunately also is a big hit with the malware folks, so occasional false-positives creep in.

**Comment by bertogravscale** [comment]
same here AVG: Win64:Evo-gen [Trj] inside mitmdump.exe

**Comment by Patlatus** [comment]
Downloaded in windows. Still blocked by AVG as threat. Hasn't anybody reported this to AVG as false positive? 
Is this link https://downloads.mitmproxy.org/11.0.0/mitmproxy-11.0.0-windows-x86_64-installer.exe not genuine? It seems to be legit

**Reply by mhils** [reply]
Yes, mitmproxy.org is the correct domain. The link you have is legit.
